multicluster/charts/linkerd-multicluster/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=1.21.0-0"
 name: "linkerd-multicluster"
 sources:
 - https://github.com/linkerd/linkerd2/
-version: 30.3.1-edge
+version: 22.10.3-edge
```

Comment:
I just want to double-check: that is 30.3.2-edge and not 30.4.2-edge as in some of the other charts? I don't know where these version numbers come from which is why I ask.

Reply:
`30.3.2-edge` is correct. The current logic behind these versions is just that we bump them, as explain in the releases doc. As for why it's something like `30.x.x`, it's because they used to be tied to the linkerd version, like for linkerd `edge-21.x.y` the chart version would be `21.x.y` (dropping the prefix because it needs to be semver), and then when we changed the versioning scheme and detached these versions from linkerd's, we bumped the major up to 30...

Reply:
Thanks for your helpful and thorough review, I believe I've integrated all of this feedback. I went through the release doc again and also double-checked that `/bin/helm-docs` and `go test ./... -output` generated no uncommited changes.